dsls/sql/pom.xml

```diff
@@ -22,7 +22,7 @@
   <parent>
     <groupId>org.apache.beam</groupId>
     <artifactId>beam-dsls-parent</artifactId>
-    <version>0.7.0-SNAPSHOT</version>
+    <version>0.6.0</version>
```

Comment:
Use current version for the parent pom, but instead overwrite dependencyManagement section they way you need it.